cmd/prchecker/main.go

```diff
@@ -41,79 +71,163 @@ func main() {
 
 	client := github.NewClient(nil)
```

Comment:
I think the easiest way to test this, is just to create an interface that can be used in place of the github client and then create a stub that you can use in tests. You might need to make the `main` function smaller to make the testing easier.